src/lwc/rd2StatusMappingSettings/rd2StatusMappingSettings.html

```diff
@@ -0,0 +1,95 @@
+<template>
+
+    <template if:true={hasMessage}>
+        <div class="slds-p-left_large slds-p-right_large" tabindex="-1">
+            <div class="desktop forcePageError" aria-live="assertive">
```

Comment:
What does the `desktop` and `forcepageError` classes do here?

Reply:
The setting was copied from the [`utilPageLevelMessage`](https://github.com/SalesforceFoundation/NPSP/blob/master/src/lwc/utilPageLevelMessage/utilPageLevelMessage.html).I cannot reused the component since we need messages to behave like a toast message not a page level message. `ShowToastEvent` cannot be used in Lightning Out.